scraper.py

```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
+postUrl = 'http://localhost:5000/tags' #LOCAL
+
+def get(url):
+    jsn = json.load(urllib2.urlopen(url))
+    items = jsn['data']
+    ret = []
+    for item in items:
+        lat = item['latitude']
+        lon = item['longitude']
+        desc = item['description']
+        name = item['outlet_name']
+        cat = "Food"
+        image = item['logo']
+        subcat = None
+
+        data = {
+            'latitude': lat,
+            'longitude': lon,
+            'name': name,
+            'description': desc,
+            'category': cat,
+            'image': image
```

Comment:
Can you make all entries generated for uwapi have the username "uwapi", so updating and managing them all is easier. I noticed right now, it's null.